### ISSUE TITLE
Not very responsive when fixing the issues

### ISSUE THREAD

**Issue by hastef88** [42Crunch/vscode-openapi#43]
Unless I run the audit tool, I cannot be sure that the issue is fixed since the error is still highlighted until I press audit button.. maybe an improvement

**Comment by ak1394** [comment]
Yes, indeed the errors are retained until the audit is run again. This is an improvement we'd like to address in a future versions of the extension.
